pkg/collector/process.go

```diff
@@ -56,7 +56,8 @@ type CollectingProcess struct {
 }
 
 type CollectorInput struct {
-	Address       net.Addr
+	HostPort      string
+	Protocol      string
```

Comment:
We can add comments here to specify the format of HostPort and Protocol. 
Also, the name `HostPort` is a little ambiguous to me, I would wonder whether there is a hostIP for the hostPort. Maybe we can keep the Address or Add as Antrea side is using `externalFlowCollectorAddr` and `collectorAddr`.

Reply:
Followed the format in net.JoinHostPort method. I can see how this can be confusing, so changed the name to Address.